apps/src/music/globals.js

```diff
@@ -25,5 +26,13 @@ export default {
 
   getPlayer() {
     return playerRef;
+  },
+
+  setProgramSequencer(programSequencer) {
+    programSequencerRef = programSequencer;
+  },
+
+  getProgramSequencer() {
+    return programSequencerRef;
```

Comment:
I know this has already been merged, but just looking back from the next PR, does ProgramSequencer need to be in Globals?

Reply:
Oh good catch; in the end it didn't need to be.  Will fix in the next PR.